src/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import "github.com/gin-gonic/gin"
+
+func main() {
+	r := gin.Default()
+	r.GET("/hello", func(c *gin.Context) {
+		c.JSON(200, gin.H{
+			"message": "Hello world!",
+		})
+	})
+	r.GET("/greet/:name", func(c *gin.Context) {
+		name := c.Param("name")
```

Comment:
Can you create a package-visible constant for the parameter name?
```suggestion
		name := c.Param(NameParam)
```